pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -234,7 +234,7 @@ func (r *ReconcileInfinispan) deploymentForInfinispan(m *infinispanv1.Infinispan
 						Image: imageName,
 						Name:  "infinispan",
 						Args: []string{configPath, "-Djboss.default.jgroups.stack=dns-ping",
-							"-Djgroups.dns_ping.dns_query=" + m.ObjectMeta.Name + "-ping.default.svc.cluster.local"},
+							"-Djgroups.dns_ping.dns_query=" + m.ObjectMeta.Name + "-ping." + m.Namespace + ".svc.cluster.local"},
```

Comment:
Isn't `Namespace` also hidden under ObjectMeta?

Reply:
no, actually I need to check

Reply:
aha anonymous field... I love the modern languages!